src/main/java/org/jenkinsci/plugins/envinject/service/BuildCauseRetriever.java

```diff
@@ -94,4 +108,21 @@ private static String getTriggerName(Cause cause) {
         return null;
     }
 
+    private static Boolean isUserCause(Cause cause) {
+    	if (Cause.UserIdCause.class.isInstance(cause)) {
```

Comment:
This code should also support the deprecated <code>UserCause</code>, because it's still being used in Hudson-compatible plugins